src/main/fc/fc_msp.c

```diff
@@ -306,6 +306,10 @@ void initActiveBoxIds(void)
         activeBoxIds[activeBoxIdCount++] = BOXANTIGRAVITY;
     }
 
+    if (!feature(FEATURE_DYNAMIC_FILTER)) {
```

Comment:
If it is not enabled as a feature, the mode becomes available. No need for the mode, if it is already enabled as a feature.
Same as airmode.